liquigraph-core/src/test/java/org/liquigraph/core/io/lock/LockableConnectionTest.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.liquigraph.core.io.lock;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.sql.Connection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.inOrder;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class LockableConnectionTest {
+
+    @Test
+    public void disables_auto_commit() throws Exception {
+        Connection delegate = mock(Connection.class);
+
+        LockableConnection lockableConnection = new LockableConnection(delegate, new LiquigraphLock());
+
+        assertThat(lockableConnection.getAutoCommit()).isFalse();
```

Comment:
You're not asserting anything with that line, other that a Mockito mock will return `false` by default for a method returning a boolean. The mock will not store the state set by `setAutoCommit(false)`.